### DISCUSSION TITLE
Error in downloading dataset through ensaio

### DISCUSSION THREAD

**Discussion by goudan-0729** [fatiando/community#156] [Q&A]
When using:
**ensaio.fetch (name of dataset)**
There are always couldn't downloading the dataset sucessfully. My soultion is downloading the dataset from the source web and put them in the **\ensaio\Cache\v1\** every time, how to solve this problem?

**Comment by leouieda** [comment]
@goudan-0729 could you please post the error message that you get and the exact code that was run?
